qa/common/infer_util.py

```diff
@@ -65,7 +65,10 @@ def _prepend_string_size(input_values):
             # don't convert it to str as Python will encode the
             # bytes which may distort the meaning
             if obj.dtype.type == np.bytes_:
```

Comment:
Isn't the numpy dtype "object"?

Reply:
Nope, the dtype for a numpy array of bytes need not be np.object. The bytes() representation is a sequence of bytes